gamemodes/terrortown/gamemode/shared/lang/de.lua

```diff
@@ -1349,3 +1349,16 @@ L.karma_enemyhurt_tooltip = "Schaden"
 L.karma_cleanround_tooltip = "Saubere Runde"
 L.karma_roundheal_tooltip = "Rundenheilung"
 L.karma_unknown_tooltip = "Unbekannt"
+
+-- 2021-05-07
+L.header_random_shop_administration = "Zufalls-Shop"
+L.header_random_shop_value_administration = "Balance Einstellungen"
+
+L.shopeditor_name_random_shops = "Aktiviere Zufalls-Shop"
+L.shopeditor_desc_random_shops = "Zufalls-Shops geben jedem Spieler eine zufällige Menge an Ausrüstung.\nTeam-Shops zwingen alle Spieler aus einem Team zu ein und demselben Shopinhalt.\nAuswürfeln erlaubt es ein gegebenes Set für Credits durch ein neues Zufallsset auszutauschen."
```

Comment:
I think this should work like this, same holds true for the other languages
```suggestion
L.shopeditor_desc_random_shops = [[Zufalls-Shops geben jedem Spieler eine zufällige Menge an Ausrüstung.
Team-Shops zwingen alle Spieler aus einem Team zu ein und demselben Shopinhalt.
Auswürfeln erlaubt es ein gegebenes Set für Credits durch ein neues Zufallsset auszutauschen.]]
```

Reply:
Oh gotta test this. Looks way better.